Adyen/HttpClient/HttpWebRequestWrapper.cs

```diff
@@ -167,53 +162,41 @@ public string Post(string endpoint, Dictionary<string, string> postParameters, C
             return new StreamReader(response.GetResponseStream()).ReadToEnd();
         }
 
-        public HttpWebRequest GetHttpWebRequest(string endpoint, Config config, bool isApiKeyRequired, RequestOptions requestOptions = null)
+        public HttpWebRequest GetHttpWebRequest(string endpoint, bool isApiKeyRequired, RequestOptions requestOptions = null)
         {
             //Add default headers
             var httpWebRequest = (HttpWebRequest)WebRequest.Create(endpoint);
```

Comment:
The purpose of this PR was to remove the `HttpWebRequest`

Reply:
I noted this in the description
> Rename HttpUrlConnectionClient to HttpWebRequestWrapper and use this implementation for backward compatibility

I left this for backward compatibility, so the users of the library can use the old approach if they are stick to HttpWebRequest.